WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -214,21 +213,23 @@ final public class InteractiveNotificationsHandler : NSObject
         case CommentReplyWithLike   = "replyto-like-comment"
 
         var actions : [NoteActionDefinition] {
-            return self.dynamicType.actionsMap[self] ?? [NoteActionDefinition]()
+            switch self {
+            case CommentApprove:
+                return [.CommentApprove]
+            case CommentLike:
+                return [.CommentLike]
+            case CommentReply:
+                return [.CommentReply]
+            case CommentReplyWithLike:
+                return [.CommentLike, .CommentReply]
+            }
```

Comment:
Thanks Eric! Definitely a waaaay better practise, since adding a new Enum Case would trigger a compiler warning. Swift takes a while to get used to, i guess!